skyportal/models.py

```diff
@@ -242,11 +265,15 @@ class Comment(Base):
     source_id = sa.Column(sa.ForeignKey('sources.id', ondelete='CASCADE'),
                           nullable=False, index=True)
     source = relationship('Source', back_populates='comments')
+    candidate_id = sa.Column(
+        sa.ForeignKey("sources.id", ondelete="CASCADE"), nullable=True, index=True
+    )
+    candidate = relationship("Source", back_populates="candidate_comments")
 
 
 class Photometry(Base):
     __tablename__ = 'photometry'
-    observed_at = sa.Column(ArrowType) # iso date
+    observed_at = sa.Column(ArrowType)  # iso date
     mjd = sa.Column(sa.Float)  # mjd date
     time_format = sa.Column(sa.String, default='iso')
     time_scale = sa.Column(sa.String, default='utc')
```

Comment:
Now that we're updating the tables via API, I think we can safely  remove `time_format` and `time_scale`. Instead, at the database level, we can *assume* that `mjd` is `utc` and the time format is `iso`. For now, the API call to post data can say in the schema that those are the formats we're assuming. In the future, we can add optional parameters `time_format` and `time_scale` (with defaults to `iso` and `mjd`) and we can do the math needed to convert to `iso` and `mjd` before updating the table.

Reply:
Sure, especially since we're already converting all times to iso/tcb in the handler: https://github.com/skyportal/skyportal/blob/master/skyportal/handlers/api/photometry.py#L61  Let's take care of this in another PR, though.